capstone/capdb/tasks.py

```diff
@@ -160,3 +161,35 @@ def get_court_count_for_jur(jurisdiction_id):
     }
 
     return results
+
+
+def create_case_text_for_all_cases(update_existing=False):
+    """
+        iterate through all volumes, call celery task for each volume
+    """
+    query = CaseMetadata.objects.all()
+
+    # if not updating existing, then only launch jobs for volumes with unindexed cases:
+    if not update_existing:
+        query = query.filter(case_text=None).distinct()
+
+    # launch a job for each volume:
+    for cmd_id in query.values_list('pk', flat=True):
```

Comment:
I'm curious whether this will work, adding 6.5 million separate jobs to celery. Maybe it's fine -- let's try it! Definitely reads more cleanly than doing it by volume, though probably runs a bit slower with more setup and teardown.

Reply:
Word. If it doesn't work it's easy enough to put them into blocks of jurisdictions or something.